src/relation.md

```diff
@@ -6,108 +6,115 @@
+
+The relation $\mathcal{R}_{\mathsf{plonkish}}$ contains pairs of instances $\mathsf{instance}$, and witnesses $w$.
+
 We say that $\mathsf{instance}$ is a valid instance whenever there exists some advice $w$ such that $(\mathsf{instance}, w) \in \mathcal{R}_{\mathsf{plonkish}}.$
 The Plonkish language $\mathcal{L}_{\mathsf{plonkish}}$ contains all valid instances.
 
-The relation $\mathcal{R}_{\mathsf{plonkish}}$ takes, as public inputs, instances of the form
-$$
-\mathsf{instance} = (\mathbb{F}, \phi, f, \equiv_A, S_I, \{ \mathsf{CUS}_{u}, p_u \}_u, \{ \mathsf{LOOK}_v, \mathsf{TAB}_v \}_v)
-$$
+If the proof system is knowledge sound, then the prover must have knowledge of the witness in order to construct a valid proof. In the setting of a zero-knowledge proof system, the witness is private and the proof leaks no additional information about it.
+
+### Instances
 
-### Inputs into $\mathcal{R}_{\mathsf{plonkish}}$
+The relation $\mathcal{R}_{\mathsf{plonkish}}$ takes instances of the following form:
 
-| Public Inputs     | Description |
+| Instance element  | Description |
 | ----------------- | -------- |
 | $\mathbb{F}$      | A prime field. |
-| $t_I$             | Number of instance entries. |
-| $\phi$            | Instance entries $\phi : \mathbb{F}^{t_I}$. |
+| $t$               | Number of instance entries. |
+| $\phi$            | Instance entries $\phi \mathrel{⦂} \mathbb{F}^t$. |
 | $n > 0$           | Number of rows. |
-| $S_I$             | A set $S_I \subseteq ([0,m_A) \times [0,n)) \times [0,t_I)$ indicating which instance entries must be used in the advice. |
-| $m_F$             | Number of fixed columns. |
-| $f$               | Fixed columns $f : \mathbb{F}^{m_F \times n}$. |
-| $S_F$             | A set $S_F \subseteq ([0,m_A) \times [0,n)) \times ([0,m_F) \times [0,n))$ indicating which fixed entries must be used in the advice. |
-| $m_A > 0$         | Number of advice columns. |
-| $\equiv_A$        | An equivalence relation on $[0,m_A) \times [0,n)$ indicating which advice entries are equal. |
-| $\mathsf{CUS}_u$  | Sets $\mathsf{CUS}_u \subseteq [0,n)$ indicating which rows the custom functions $p_u: \mathbb{F}^{m_F + m_A} \mapsto \mathbb{F}$ are applied to. |
-| $p_u$             | Custom multivariate polynomials $p_u: \mathbb{F}^{m_F + m_A}  \mapsto \mathbb{F}$. |
+| $m$               | Number of columns. |
+| $\equiv$          | An equivalence relation on $[0,m) \times [0,n)$ indicating which advice entries are equal. |
+| $S$               | A set $S \subseteq ([0,m) \times [0,n)) \times [0,t)$ indicating which instance entries must be used in the advice. |
+| $m_f \leq m$      | Number of columns that are fixed. |
+| $f$               | The fixed content of the first $m_f$ columns, $f \mathrel{⦂} \mathbb{F}^{m_f \times n}$. |
+| $\mathsf{CUS}_u$  | Sets $\mathsf{CUS}_u \subseteq [0,n)$ indicating which rows the custom functions $p_u \mathrel{⦂} \mathbb{F}^{m} \mapsto \mathbb{F}$ are applied to. |
```

Comment:
Does it make sense to call them "custom" constraints, given that there are no "standard" constraints in this model? Perhaps we should call them something like "gate constraints".